### PR TITLE
Modify UX/Output

### PR DESCRIPTION
**Pull request by katherinelc321**
https://issues.redhat.com/browse/OSD-5676

### CODE REVIEW

File: cmd/account/mgmt/account-list.go [hunk 1]
```diff
@@ -19,12 +21,24 @@ type accountListOptions struct {
...
 	payerAccount string
 	accountID    string
+	output       string
 
 	flags      *genericclioptions.ConfigFlags
 	printFlags *printer.PrintFlags
 	genericclioptions.IOStreams
 }
 
+type listResponse struct {
```
[the remark this answers is not in the document]
Comment (katherinelc321) [reply cmd/account/mgmt/account-list.go:31]: i've tested this locally without the additional struct and it does list accounts from several devs

**Review by iamkirkbater — APPROVED** [review]
Awesome, let's just squash this down to a single commit and we can merge this!

**Comment by fahlmant** [comment]
@katherinelc321 Please wait for the bot to merge PRs

**Comment by georgettica** [comment]
@iamkirkbater , didn't squash in the end :(

**Comment by fahlmant** [comment]
Can we revert this and squash? Or fix up the commit messages?

**Pull request merged**